README.md

```diff
@@ -15,6 +15,8 @@ Express, PG, EJS, CSS, JavaScript, HTML, Node.
 ## Change Log
 
 18 Sept 1154 -- project fully operational.

```

Comment:
make sure you preview your markdown files, these 2 lines are on the same line without a page break (you could make it a list by adding `- ` in front of each line)